inc/Editor/BlockManagement/BlockRegistration.php

```diff
@@ -49,8 +49,20 @@ public static function register_helper_blocks(): void {
 		// Remote data pagination block - used to render pagination links for collections.
 		register_block_type( REMOTE_DATA_BLOCKS__PLUGIN_DIRECTORY . '/build/blocks/remote-data-pagination' );
 
-		// Remote data empty result block - used to render a message when no results are found.
-		register_block_type( REMOTE_DATA_BLOCKS__PLUGIN_DIRECTORY . '/build/blocks/remote-data-no-results' );
+		// Remote data error fallback block - used to render a message when results aren't available to load.
+		register_block_type( REMOTE_DATA_BLOCKS__PLUGIN_DIRECTORY . '/build/blocks/remote-data-no-results', [
+			'variations' => [
+				[
+					'name' => 'remote-data-blocks/error',
+					'title' => 'Error',
+					'description' => 'Display an error message when the remote data fails to load.',
+					'attributes' => [
+						'mode' => 'error',
+					],
+					'isActive' => [ 'mode' ],
+				],
+			],
+		] );
```

Comment:
I tested with this and works fine.

```suggestion
		register_block_type( REMOTE_DATA_BLOCKS__PLUGIN_DIRECTORY . '/build/blocks/remote-data-no-results' );
```

Reply:
It hadn't worked before, but I am guessing the JS registration or the `createBlock` using the variation worked.